### DISCUSSION TITLE
Is the S1300i network-enabled?

### DISCUSSION THREAD

**Discussion by tinloaf** [jonaswinkler/paperless-ng#967] [General]
Hi,

[the documentation](https://paperless-ng.readthedocs.io/en/latest/scanners.html) claims that the Fujitsu S1300i can scan to FTP. However, the specs I can find say nothing about the scanner having some kind of network connectivity - and I would expect Fujitsu to pretty loudly advertise this feature.

What's the truth here? Can the scanner scan to FTP or not?

Thanks a lot,
Lukas

**Comment by jonaswinkler** [comment]
I just looked into the software and couldn't find an option to scan to FTP. I've only ever used the scan to folder option.

**Reply by tinloaf** [reply]
Thanks for the info. Maybe I'll still get it and just put a raspberry pi next to it that does the scan-and-upload thing.

**Reply by jonaswinkler** [reply]
There's some option to execute arbitrary programs (with the scanned filename as an argument, probably), so I figure it should be possible.

![image](https://user-images.githubusercontent.com/17569239/116756668-c4062a80-aa0c-11eb-9d23-1c2f03669d8b.png)
![image](https://user-images.githubusercontent.com/17569239/116756700-d6806400-aa0c-11eb-8c4f-cb1eefd4ae87.png)


Apart from that, 3000 scanned sheets, no issues so far, and reasonably fast for a consumer device.

**Reply by jonaswinkler** [reply]
You could probably also write a small program that simply takes a file, and uploads that using the Paperless REST API.

**Reply by tinloaf** [reply]
Thanks for the recommendation! The software looks Windows-only, though, right? So, as a Linux user, I'm probably stuck with cobbling together myself - but having a script that just performs a single scan and then feeds a REST API sounds simple enough.

**Reply by jonaswinkler** [reply]
On linux there's `gscan2pdf` - The scanner works perfectly fine. The hardest part was searching for some proprietary firmware, but after that, it worked just as good as on windows. However, it took a lot more effort on linux to get certain things right, such as contrast and brightness for black/white scans. The windows software produces pretty good PDF documents out of the box.

**Reply by jonaswinkler** [reply]
Oh, and I never got the physical "Scan Now" button to work on linux, which is very handy.

**Reply by tinloaf** [reply]
Hi @jonaswinkler ,

one more follow-up to the S1300i. I got one, set up paperless-ng (great docker-compose setup workflow, by the way :+1: ), everything is working well - except I'm underwhelmed by the scanning quality. It produces very "stripy" results, see this example: 

https://cloud.tinloaf.de/s/dgTYDbLTMCiLpFb

Especially on the front you have a rather bright lane in the middle third of the page, and less brightness on the rest of the page. I used `scanimage --dev epjitsu:libusb:001:006 --source 'ADF Duplex' --mode 'Color' --page-width 211 --page-height 294 --batch --resolution 150` to get that scan.

I assume this is problem of uneven illumination. Now I wonder if that is just something I must live with if I want to have a scanner for less than 400 €, or if my ScanSnap is defective and I should return it. Or is this rather just something that one needs to address using some post-processing?

Does your S1300i produce more even results? Thanks for your help!

**Reply by jonaswinkler** [reply]
Same command as above.

![out1](https://user-images.githubusercontent.com/17569239/118376712-fd709580-b5c9-11eb-83df-3ac6a2c69c6e.jpg)

Never observed these artifacts with the scanner software from Jujitsu though. I rarely use the scanner on linux. I scan most of the documents in pure black/white, and only some in greyscale.

**Reply by tinloaf** [reply]
Thanks! Then it looks like this is just a technical limitation of this scanner. I guess I can live with it.
